components/Banner/ExitableBanner.tsx

```diff
@@ -27,13 +29,16 @@ export default function ExitableBanner({
   exitButtonPositionOverride,
   onExit,
 }: Props): ReactElement | null {
-  const [isExited, setIsExited] = useState<boolean>(
-    getCookieOrLocalStorageValue({
-      key: bannerKey,
-    })?.value === "exited"
-  );
+  const auth = useSelector((state: RootState) => state.auth);
+  
+  const {
+    isDismissed,
+    dismissFeature,
+    dismissStatus
+  } = useDismissableFeature({ auth, featureName: bannerKey })
+  
 
-  if (isExited) {
+  if (dismissStatus === "unchecked" || (dismissStatus === "checked" && isDismissed)) {
```

Comment:
lol my first reaction to this was "wuut" haha can you explain the second part of the conditional?

Reply:
Ah right, `isDismissed` is `false` by default until `dismissStatus`  is fetched from server.

Basically the component does not render the `ExitableBanner` until confirmed by the server that user has not dismissed it